src/net_processing.cpp

```diff
@@ -3808,6 +3810,30 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
                     }
                 }
 
+                // use mockable current_time rather than nNow to allow for testing
+                const auto current_time = GetTime<std::chrono::microseconds>();
+
+                // Check for rebroadcasts
+                if (pto->m_next_rebroadcast < current_time) {
+                    LogPrint(BCLog::NET, "Rebroadcast timer triggered\n");
+                    // schedule next rebroadcast
+                    bool fFirst = (pto->m_next_rebroadcast.count() == 0);
```

Comment:
I'd rather leave it in for legibility since the compiler should optimize it away regardless- (simple example reproduced [here](https://godbolt.org/z/ZcVdJK)). If invoked, the `fSkipRun` logic needs to overwrite the next rebroadcast time, so that logic would also need to be shuffled. Doable but makes the code more opaque.